.github/workflows/posix.yml

```diff
@@ -62,8 +59,27 @@ jobs:
             pandas_testing_mode: "deprecate"
             test_args: "-W error"
             name: "Numpy Dev"
+          # Test lowest & highest supported pyarrow
+          # Don't test pyarrow v2/3: Causes timeouts in read_csv engine
+          # even if tests are skipped/xfailed
+          - env_file: actions-38.yaml
+            pattern: "not single_cpu"
+            pyarrow_version: 5
+            name: "Min Pyarrow"
```

Comment:
Since the min pyarrow is actually 1.0.1, better just to name this like "Pyarrow v5"